MaterialDesignThemes.Wpf/SnackbarMessageQueueItem.cs

```diff
@@ -47,8 +48,37 @@ internal class SnackbarMessageQueueItem
         public bool IsPromoted { get; }
 
         /// <summary>
-        /// Still display this message even if it is a duplicate.
+        /// Always show this message, even if it's a duplicate
         /// </summary>
-        public bool IgnoreDuplicate { get; }
+        public bool ShowAlways { get; }
```

Comment:
I am wondering about changing the order of this to be "AlwaysShow". IMO it reads a little nicer (like the comment above it)

Reply:
Agreed.